protobuf/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 5.0.0-wip
 
+* Improve performance of `GeneratedMessage.deepCopy`. ([#742])
```

Comment:
We should update the pubspec version to agree w/ the changelog (=> `5.0.0-wip`). If we do plan to publish after landing this (no particular opinion), we can change both versions to `5.0.0`).

Reply:
Didn't we discuss this already? https://github.com/google/protobuf.dart/pull/1037#discussion_r2258129449

5.0.0-wip breaks the build because it's not considered 5.0.0, and plugin now needs 5.0.0.

We can update the changelog before or even after releasing 5.0.0.